### PR TITLE
chore(samples/pytorch) - Build script fix - Updating Dockerfile name

### PR DESCRIPTION
**Pull request by shrinath-suresh**
https://github.com/kubeflow/pipelines/pull/5863 -  We no longer have separate dockerfile for cpu and gpu. The build script needs to be updated accordingly.

In this PR, dockerfile name has been removed from the `docker build` argument.

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Review by jagadeeshi2i — APPROVED** [review]

File: samples/contrib/pytorch-samples/bert/config.properties [hunk 1]
```diff
@@ -4,7 +4,7 @@ metrics_address=http://0.0.0.0:8082
 enable_metrics_api=true
 metrics_format=prometheus
 number_of_netty_threads=4
-install_py_dep_per_model=true
+install_py_dep_per_model=false
```
Comment (shrinath-suresh) [inline samples/contrib/pytorch-samples/bert/config.properties:7]: Setting it to `false` until Custom package installation issue is fixed in torchserve

File: samples/contrib/pytorch-samples/build.sh [hunk 2]
```diff
@@ -25,7 +25,7 @@ export full_image_name=$full_image_name
 ## build and push docker - to fetch the latest changes and install dependencies
 # cd pytorch_kfp_components
 
-docker build --no-cache -t $full_image_name -f Dockerfile-cpu .
```
Comment (shrinath-suresh) [inline samples/contrib/pytorch-samples/build.sh:28]: We no longer need the dockerfile name as we have only one dockerfile now for both cpu and gpu

Comment (jagadeeshi2i) [reply samples/contrib/pytorch-samples/build.sh:28]: /lgtm

**Pull request merged**